### PR TITLE
Responsive breakpoints

### PR DESCRIPTION
**Pull request by mackenziekerwin**
I styled a few components on the homepage to demonstrate the responsive functionality but they are far from finished--this is just to get a general idea of the breakpoints.

### Example:
Figma: desktop
Screen Shot 2022-04-05 at 7 56 44 PM

Figma: tablet
Screen Shot 2022-04-05 at 7 57 05 PM

Figma: mobile
Screen Shot 2022-04-05 at 7 57 28 PM

p.s. should we include a `max-width` in our `<MarginContainer>` for super large desktops?

### CODE REVIEW

File: src/components/ProjectPreview/index.js [hunk 1]
```diff
@@ -10,33 +19,41 @@ const ProjectPreview = ({ project }) => {
...
-              <h2>{title.text}</h2>
-              <Link to={`/works/${uid}`}>Watch</Link>
-            </div>
-          );
-        })}
-      </Content>
+            } = work;
+            return (
+              <div>
+                <img src={thumbnail_url} />
```
Comment (mnav0) [inline src/components/ProjectPreview/index.js]: need alt text here! you could leverage the title field and set the alt text using that

File: src/styles/styles.js [hunk 2]
```diff
@@ -1,53 +1,60 @@
...
-  font-family: ${fonts.epilogue};
-  padding-left: 5px;
-  padding-right: 5px;
+  padding: 0.5rem 1.5rem;
   transform: skew(15deg);
-  color: ${CREAM};
+  color: ${(props) => (props.color === YELLOW ? BLACK : CREAM)};
   border-radius: 2px;
   border: 2px solid ${(props) => (props.color ? props.color : BLUE)};
   background-color: ${(props) => (props.color ? props.color : BLUE)};
```
Comment (mnav0) [inline src/styles/styles.js:60]: add a `cursor: pointer`

**Review by mnav0 — COMMENTED** [review]
looks great ! a few comments

**Review by mnav0 — APPROVED** [review]
lgtm! i just saw your comment about a `max-width`, i think that would help you avoid having to do a lot of extra styling for larger screens within the components themselves so i think that would be a good idea to add

**Review by gabbybruck — APPROVED** [review]

**Pull request merged**